bin/stripe-pipeline

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env python
+
+from stripe_pipeline.cli import cli
```

Comment:
A tiny wrapper executable for the `cli` module

Reply:
I think this file could be deleted if the setup script uses the [`entrypoint` parameter](http://click.pocoo.org/5/setuptools/#introduction). The entrypoint specifies the name of the binary package and which function should it reference. 

In this case could be something like:

```python
entry_points={
    'console_scripts': [
        'stripe-pipeline=stripe_pipeline.cli.cli'
    ]
}
```

Reply:
Whoa, didn't know about that! I'll try that out!